Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.yml

```diff
@@ -0,0 +1,1208 @@
+    - contextPath: SekoiaXDR.ListAlerts.rule.severity
+      description: The severity level of the rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.rule.type
+      description: The type of rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.rule.uuid
+      description: The unique identifier of the rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.rule.name
+      description: The name of the rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.rule.description
+      description: The description of the rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.rule.pattern
+      description: The pattern of the rule that triggered the alert.
+    - contextPath: SekoiaXDR.ListAlerts.similar
+      description: The number of similar alerts to this one.
+    - contextPath: SekoiaXDR.ListAlerts.status.name
+      description: The name of the status of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.status.description
+      description: The description of the status of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.status.uuid
+      description: The unique identifier of the status of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.urgency.criticity
+      description: The level of criticity of the urgency of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.urgency.current_value
+      description: The current value of the urgency of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.urgency.severity
+      description: The severity level of the urgency of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.urgency.display
+      description: The display of the urgency of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.urgency.value
+      description: The value of the urgency of the alert.
+    - contextPath: SekoiaXDR.ListAlerts.created_by
+      description: The user who created the alert.
+    - contextPath: SekoiaXDR.ListAlerts.number_of_total_comments
+      description: The total number of comments on the alert.
+    - contextPath: SekoiaXDR.ListAlerts.time_to_detect
+      description: The time it took to detect the alert.
+    - contextPath: SekoiaXDR.ListAlerts.entity.name
+      description: The name of the entity associated with the alert.
+    - contextPath: SekoiaXDR.ListAlerts.entity.uuid
+      description: The unique identifier of the entity associated with the alert.
+    - contextPath: SekoiaXDR.ListAlerts.created_by_type
+      description: The type of user who created the alert.
+    - contextPath: SekoiaXDR.ListAlerts.details
+      description: The details of the alert.
+    description: Command to retrieve a list of Alerts from Sekoia XDR.
+  - name: sekoia-xdr-get-alert
+    arguments:
+    - name: id
+      required: true
+      description: 'The uuid or short_id of the alert to retrieve. i.e: "f5dcb81c-8d81-4332-9f1e-f119a1b31217" or "ALUnyZCYZ9Ga".'
```

Comment:
If is recommended to add to the argument description which command to use in order to retrieve the ID. In this case the relevant command is probably sekoia-xdr-list-alerts.